src/devices/sonoff.ts

```diff
@@ -1125,6 +1126,14 @@ const definitions: Definition[] = [
                 description: 'The water valve is in normal state, water shortage or water leakage',
                 access: 'STATE_GET',
             }),
+            binary({
+                name: 'behavior_of_water_shortage',
```

Comment:
I think we can use a more self explaining name here, something like `auto_close`?

Reply:
> I think we can use a more self explaining name here, something like `auto_close`?

how about 'auto_close_when_water_shortage', our product manager thinks it tells users more accurately.

Reply:
Sounds good